eZ/Publish/Core/SignalSlot/Tests/UserServiceTest.php

```diff
@@ -91,13 +91,25 @@ public function serviceProvider()
             ),
             array(
```

Comment:
Note: on purpose keeping long array syntax to avoid obfuscating actual changes :P 
I can convert entire block of these in a separate commit after reviews.